### ISSUE TITLE
Error in final slope of Narrabeen & Cable Beach example

### ISSUE THREAD

**Issue by DanieTheron** [kvos/CoastSat.slope#20]
Hi,
I have loaded both examples and get a similar false output - my slopes are the same for each transect, and also, it is the upper bound that was specified (0.2). I have not changed any of the code in the example file; therefore, I do not understand where this error could have come in. I believe this might be a similar issue to #18 or #17 however, I could not spot an error (especially since I have not changed the original example code). Do you perhaps have any idea why this might occur @kvos ?

My code & Output:
[Example_Error_slope_Narrabeen.pdf](https://github.com/kvos/CoastSat.slope/files/9277210/Example_Error_slope_Narrabeen.pdf)

**Comment by kvos** [comment]
interesting... that's just running the examples as they are? I will check it out and get back to you

**Comment by DanieTheron** [comment]
Yes, running the examples just as they are - no changes to CoastSat or the jupyter notebook example.

**Comment by DanieTheron** [comment]
Yes, running the examples just as they are - no changes to CoastSat or the jupyter notebook example.

**Comment by DanieTheron** [comment]
Hi Killian @kvos ,

Just wanted to say I found the problem (and my slopes work now! yay). So when running the example code for Narrabeen and Cable beach with the sitename_tide.pkl files provided it works. But when I ran the FES2014 model, it did not work. Upon review of the exact content of the .pkl files by converting it to a .csv to see what the format and contents are; I saw that the FES2014 data had an additional column, and the tides were scaled with a factor of 10 (when compared to measured tides at my site of interest). So I manipulated the FES2014 data to match the measured tides and saved it to a .csv file in the same format as the .pkl files of the examples. Within the code, I extracted the tide data from the .csv to a .pkl (in the same format as yours) - and it works.

Thanks for your great work (CoastSat & CoastSat.slope)!
